src/gui/autosave_dialog.h

```diff
@@ -57,7 +57,7 @@ Q_OBJECT
 	 * @param actual_path    The path which is currently selected.
 	 * @param parent         The parent window.
 	 */
-	AutosaveDialog(const QString& original_path, const QString& autosave_path, const QString& actual_path, MainWindow* parent = nullptr, Qt::WindowFlags f = 0);
+	AutosaveDialog(const QString& original_path, const QString& autosave_path, const QString& actual_path, MainWindow* parent = nullptr, Qt::WindowFlags f = Qt::WindowFlags());
```

Comment:
I'm fine with the proposed form. But usually I avoid spelling the type twice, by using `... = {}`

Reply:
Thanks for the review. I'll adjust the line and merge the changes.

Reply:
Changed all the occurrences to value initialization syntax and updated the commit message to reflect the fact that the code follows Mapper convention instead of the Qt style.